code/game/objects/items/weaponry.dm

```diff
@@ -470,6 +470,18 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "A bust of the famous Greek physician Hippocrates of Kos, often referred to as the father of western medicine."
 	icon_state = "hippocratic"
 
+/obj/item/statuebust/attack_self(mob/living/user)
+	user.examinate(src)
+
+/obj/item/statuebust/examine(mob/living/user)
+	. = ..()
+	if(.)
+		return
+	add_fingerprint(user)

```

Comment:
You're adding fingerprints on a distance. Also this whole block should probably be checked against living at least so ghosts examining stuff won't trigger it.